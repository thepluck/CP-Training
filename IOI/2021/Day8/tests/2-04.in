50 100
1 28
30 48
19 18
34 36
27 5
41 9
23 11
8 38
16 22
9 10
5 4
21 32
43 11
25 21
31 12
14 13
44 2
14 15
13 12
40 45
21 8
17 40
8 9
5 37
17 33
11 10
20 30
24 2
1 17
35 29
29 2
11 47
7 8
46 37
11 12
33 34
1 7
32 42
13 20
3 4
17 18
1 39
2 3
38 49
5 6
16 15
1 2
15 26
44 50
50
1 15 18
1 29 58
2 15
1 17 8
1 9 20
1 14 20
1 11 20
1 48 0
2 9
1 50 3
1 49 2
1 22 13
1 31 0
1 19 3
2 11
2 17
1 9 28
1 44 6
1 28 2
1 35 8
1 18 4
2 9
1 15 14
2 19
1 34 2
1 47 5
1 43 1
2 15
1 40 0
1 9 26
2 31
2 29
1 38 29
1 42 14
1 11 26
1 21 18
2 34
1 25 4
1 41 0
1 23 0
2 22
1 13 20
1 20 0
1 5 1
1 32 14
2 28
2 47
1 34 4
1 3 4
2 49
