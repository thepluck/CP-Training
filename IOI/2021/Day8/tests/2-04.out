237930091
38435537
352902376
303626313
572790911
17853391
880011509
880011509
880011509
812142811
12080470
51096357
978555584
978555584
798460406
915314565
654225487
296749672
926545542
745165951
131274000
5250960
163622102
354090555
770452768
124230645
1440
45
1800
1800
1800
1800
38184929
553810635
436284944
436284944
418608000
114935979
207662400
207662400
207662400
388505394
388505394
388505394
931526572
931526572
57118122
198614811
531429293
975279986
258399377
